recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ outputs:
       requires:
         - pytest >= 4.4.*
         - nbval ==0.9.3
-        - python-graphviz >=0.8.4
+        - python-graphviz >=0.8.4,<0.16
```

Comment:
@ParthivNaresh I updated this to have a max cap at 0.16 instead of hard-coding to 0.15. LMK if you think this is a bad idea and we can discuss.